scenario01/jmeter/01-Scenario-1-SSO-multiple-heterogeneous-IDFP.jmx

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<jmeterTestPlan version="1.2" properties="3.2" jmeter="3.2 r1790748">
+<jmeterTestPlan version="1.2" properties="3.2" jmeter="3.3 r1808647">
```

Comment:
The scripts are tested with Jmeter verison 3.2 only. There were some issues we observed when running the tests in Jmeter 3.3. Therefore, can you please continue to use the same Jmeter version for now until we fully test all the scripts on a newer jmeter version?

Reply:
I've tested this with JMeter 3.3 and it got succeeded. Do we still need this change?